ivy/functional/backends/numpy/experimental/manipulation.py

```diff
@@ -483,3 +487,25 @@ def column_stack(
     arrays: Sequence[np.ndarray], /, *, out: Optional[np.ndarray] = None
 ) -> np.ndarray:
     return np.column_stack(arrays)
+
+
+@with_supported_dtypes(
+    {"1.25.2 and below": ("float32", "float64", "int32", "int64")}, backend_version
+)
+def put_along_axis(
+    arr: np.ndarray,
+    indices: np.ndarray,
+    values: np.ndarray,
+    axis: int,
+    /,
+    *,
+    mode: str = None,
+    out: Optional[np.ndarray] = None,
+):
+    np.put_along_axis(arr, indices, values, axis)
+    return ivy.inplace_update(out, arr) if ivy.exists(out) else arr
+
+
+put_along_axis.partial_mixed_handler = lambda *args, mode=None, **kwargs: mode in [
+    "assign",
+]
```

Comment:
Hi @mobley-trent, `test_put_along_axis` is passing now 💪
I just forgot to update the numpy and paddle backend with the new `mode` names, so currently only the torch backend is being used. Please update the rest and fix any issues that might come up.